share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -701,8 +681,8 @@ aws-isc-aarch64-build:
     SPACK_CI_STACK_NAME: tutorial
 
 tutorial-generate:
-  extends: [ ".tutorial", ".generate-x86_64"]
-  image: ghcr.io/spack/tutorial-ubuntu-22.04:v2023-10-30
+  extends: [ ".tutorial", ".generate"]
+  image: ghcr.io/spack/tutorial-ubuntu-22.04:v2023-05-07
```

Comment:
Ditto the arch-specific generate thing.  And don't move us back to an older image please.